asreview/feature_extraction/tfidf.py

```diff
@@ -9,21 +7,26 @@ class Tfidf(BaseFeatureExtraction):
     """Class to apply SKLearn Tf-idf to texts."""
     name = "tfidf"
 
-    def __init__(self, *args, ngram_max=1, **kwargs):
+    def __init__(self, *args, ngram_max=1, stop_words=0, **kwargs):
         """Initialize tfidf class.
 
         Arguments
         ---------
         ngram_max: int
             Can use up to ngrams up to ngram_max. For example in the case of
             ngram_max=2, monograms and bigrams could be used.
+        stop_words: int
+            Use *ENGLISH* stop words.
         """
         super(Tfidf, self).__init__(*args, **kwargs)
         self.ngram_max = ngram_max
-        self._model = Pipeline([
-            ('vect', CountVectorizer(ngram_range=(1, ngram_max))),
```

Comment:
What happened with the CountVectorizer?

Reply:
TfidfVectorizer does the same thing without needing a pipeline. Haven't run the numbers, but I'd think it is more efficient.

Reply:
Ow, thanks, I didn't notice that one is  `TfidfVectorizer` while the other is `TfidfTransformer`.